sale_order_retrieve_customer_after_delivery/i18n/es.po

```diff
@@ -0,0 +1,31 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+# 	* sale_order_retrieve_customer_after_delivery
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 17.0\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2024-10-22 06:40+0000\n"
+"PO-Revision-Date: 2024-10-22 06:40+0000\n"
+"Last-Translator: \n"
+"Language-Team: \n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: \n"
+"Plural-Forms: \n"
+
+#. module: sale_order_retrieve_customer_after_delivery
+#: model_terms:ir.ui.view,arch_db:sale_order_retrieve_customer_after_delivery.sale_order_view_form
+msgid "Delivery Address"
+msgstr "Dirección de entrega"
+
+#. module: sale_order_retrieve_customer_after_delivery
+#: model:ir.model.fields,field_description:sale_order_retrieve_customer_after_delivery.field_sale_order__partner_shipping_2_id
+msgid "Delivery Address (new)"
+msgstr "Dirección de entrega"
```

Comment:
Better this (this label is only for field definition, so we've interested in marking it different than original one)

```suggestion
msgstr "Dirección de entrega (nueva)"
```